apps/services/routers.py

```diff
@@ -179,16 +180,18 @@ async def cancel_request(
     user=Depends(get_current_user),
     session: AsyncSession = Depends(get_async_session),
 ):
-    """Method for cancel user request.
+    """**Method for cancel user request**.
 
-    Path:
-    - university_id: user university id
-    - user_request_id: user request id
+    **Path**:
+    - **university_id**: user university id
+    - **user_request_id**: user request id
 
-    Input:
-    - status_id: user request status id, required
+    **Input**:
+    - **status_id**: user request status id, required
 
-    Return: canceled user request id and status id
+    **Return**:
```

Comment:
pls update comments if you use different out schema

Reply:
ok